app/config.py

```diff
@@ -262,6 +262,11 @@ class Config(object):
                 "options": {"queue": QueueNames.PERIODIC},
             },
             # app/celery/nightly_tasks.py
+            "delete-oldest-quarter-of-unneeded-notification-history": {
+                "task": "delete-oldest-quarter-of-unneeded-notification-history",
+                "schedule": crontab(hour=21),
+                "options": {"queue": QueueNames.PERIODIC},
+            },
```

Comment:
An alternative if we're worried about workers being rolled, or disabling this when it becomes less relevant, would be to schedule this as a task via concourse. In PaaS we can run via `cf run-task` and in ECS through a similar mechanism.

Then we can just pause the pipeline when it no longer needs to run nightly, and turn it back on when we want it to start kicking again. Not that we should really need to disable it, I think.